manifests/modules/observability/base/.workshop/terraform/addon.tf

```diff
@@ -0,0 +1,118 @@
+resource "kubernetes_cluster_role" "eks-console-dashboard-full-access-clusterrole" {
```

Comment:
Is this file still needed with the logic in `use-cluster`? Will it not do the same thing? Or is there something else?

Reply:
Forgot to delete those files. I isn't needed.

Reply:
Unnecessary files removed.